### PR TITLE
Fix .gitignore and add missing vendors

### PR DESCRIPTION
**Pull request by Quentin-M**
Just built on a fresh GOPATH. Worked fine! Sorry about that.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,5 +6,9 @@
 **/ssh-key.pem
 build/**
 config.tfvars
+terraform.tfvars
+pull-secret.json
+tectonic-license.txt
 generated/
 bin/
+
```
Comment (squat) [inline .gitignore]: extra

File: .gitignore [hunk 2]
```diff
@@ -6,5 +6,9 @@
 **/ssh-key.pem
 build/**
 config.tfvars
+terraform.tfvars
+pull-secret.json
+tectonic-license.txt
 generated/
 bin/
```
Comment (ggreer) [inline .gitignore]: Won't lines 9 & 10 still match all kinds of stuff that we probably don't want to match?

**Comment by derekparker** [comment]
Just ran into this!

File: .gitignore (hunk 2 above)
Comment (Quentin-M) [inline .gitignore]: I think these names are pretty specific?

**Comment by Quentin-M** [comment]
Thanks guys

File: .gitignore (hunk 2 above)
Comment (ggreer) [inline .gitignore]: Any directory named "generated" or any directory named "bin"? I could see those existing in vendor.

Comment (squat) [inline .gitignore]: this takes care of line 3 in installer/.gitignore

File: installer/.gitignore [hunk 3]
```diff
@@ -1,9 +1,7 @@
...
-tools/
-matchbox
-tests/scripts/aws/output/
-scripts/cluster/output/
+/assets/frontend/scripts/
+/bin/
+/binassets/assets.go
+/frontend/node_modules/
+/matchbox
+/tests/scripts/aws/output/
```
Comment (squat) [inline installer/.gitignore]: I don't think any of these besides `/matchbox` should have the `/` prefix.

**Pull request merged**